samples/sample_encode/src/pipeline_encode.cpp

```diff
@@ -414,6 +414,34 @@ mfxStatus CEncodingPipeline::InitMfxEncParams(sInputParams *pInParams)
 
     if (m_mfxEncParams.mfx.RateControlMethod == MFX_RATECONTROL_CQP)
     {
+        bool enableRepackCtrl = pInParams->repackctrlFile != NULL && pInParams->CodecId == MFX_CODEC_AVC;
+
+        if (m_pRepackCtrl_in == NULL && enableRepackCtrl)
+        {
+            MSDK_FOPEN(m_pRepackCtrl_in, pInParams->repackctrlFile, MSDK_CHAR("rb"));
+            if (m_pRepackCtrl_in == NULL)
+            {
+                msdk_printf(MSDK_STRING("ERROR: Can't open file %s\n"), pInParams->repackctrlFile);
+                return MFX_ERR_UNSUPPORTED;
+            }
+            msdk_printf(MSDK_STRING("Using Frame Size control input file: %s\n"), pInParams->repackctrlFile);
+
+            SAFE_FREAD(&(m_CodingOption3.RepackMaxIFrameSize), sizeof(m_CodingOption3.RepackMaxIFrameSize), 
+                       1, m_pRepackCtrl_in, MFX_ERR_MORE_DATA);
+            SAFE_FREAD(&(m_CodingOption3.RepackMaxPFrameSize), sizeof(m_CodingOption3.RepackMaxPFrameSize), 
+                       1, m_pRepackCtrl_in, MFX_ERR_MORE_DATA);
+            SAFE_FREAD(&(m_CodingOption3.RepackMaxBFrameSize), sizeof(m_CodingOption3.RepackMaxBFrameSize), 
+                       1, m_pRepackCtrl_in, MFX_ERR_MORE_DATA);
+            SAFE_FREAD(&(m_CodingOption3.RepackNumPasses), sizeof(m_CodingOption3.RepackNumPasses), 
+                       1, m_pRepackCtrl_in, MFX_ERR_MORE_DATA);
+            SAFE_FREAD(m_CodingOption3.RepackDeltaQP, sizeof(m_CodingOption3.RepackDeltaQP[0]) * 8, 
```

Comment:
Better sizeof(m_CodingOption3.RepackDeltaQP)

Reply:
Will modify it, thanks!